app/Http/Controllers/MyController.php

```diff
@@ -120,7 +120,7 @@ public function postSelect(Request $request) {
         }
 
         if ($_POST['facility'] == "0") {
-            return redirect('/my/transfer')->with('error', "You didn't select a facility!");
+            return redirect('/my/select')->with('error', "You didn't select a facility!");
```

Comment:
Why did you change this?

Reply:
The user didn't start on the transfer page, they started on the select page. So I corrected it to return back to the page they were originally on.